doc/src/programmers-guide.md

```diff
@@ -548,6 +548,7 @@ The AtomVM I2C implementation uses the AtomVM Port mechanism and must be initial
 | `scl_io_num` | `integer()` | yes | I2C clock pin (SCL) |
 | `sda_io_num` | `integer()` | yes | I2C data pin (SDA) |
 | `i2c_clock_hz` | `integer()` | yes | I2C clock frequency (in hertz) |
+| `i2c_num` | `0 .. I2C_NUM_MAX - 1` | no (default: `0`) | I2C port number.  `I2C_NUM_MAX` is defined by the device SDK.  On ESP32, this value is 1. |
```

Comment:
The example immediately following this change does not include the I2C port number.

Reply:
Should probably update the i2c example for the sht31 too.
https://github.com/atomvm/AtomVM/blob/master/examples/erlang/esp32/sht31.erl

Reply:
That's because it is an "advanced feature" and had a reasonable default.